ffi/build.py

```diff
@@ -175,7 +175,9 @@ def main_posix(kind, library_ext):
     # Get LLVM information for building
     libs = run_llvm_config(llvm_config, "--system-libs --libs all".split())
     # Normalize whitespace (trim newlines)
-    os.environ['LLVM_LIBS'] = ' '.join(libs.split())
+    os.environ['LLVM_LIBS'] = \
+        '-llldCOFF -llldCommon -llldCore -llldDriver -llldELF -llldMachO -llldMinGW -llldReaderWriter -llldWasm -llldYAML ' + \
```

Comment:
I can't be precise about what, but there's something that looks odd about this. I do wonder if there's a better way to obtain this list other than hard-coding it? What if some of these are not present on a system?

Reply:
I can look into it. I'm unsure if its possible. 

I believe this comes from the commits in the original PR.